cli/command/mount.go

```diff
@@ -36,11 +36,13 @@ import (
 
 var (
 	mountExample = `Examples:
-  $ curveadm mount /s3_001 /path/to/mount -c client.yaml  # Mount CurveFS '/s3_001' to '/path/to/mount'`
+  $ curveadm mount /s3_001     s3     /path/to/mount -c client.yaml  # Mount a s3 CurveFS '/s3_001' to '/path/to/mount'
```

Comment:
maybe add a option for filesystem type with `s3` defaut is better, it can reduce the command complexity, like:

* Mount s3 file system:
```
curveadm mount NAME_OF_CURVEFS MOUNT_POINT
```

* Mount CurveBS file system:
```
curveadm mount NAME_OF_CURVEFS MOUNT_POINT --fstype=curvebs
```

Reply:
fix